src/main/java/com/robinpowered/sdk/model/Amenity.java

```diff
@@ -0,0 +1,102 @@
+package com.robinpowered.sdk.model;
+
+import com.google.common.base.Objects;
+import org.joda.time.DateTime;
+
+/**
+ * An amenity that may be associated to a {@link Space}.
+ *
+ * <p>
+ * Amenities may be created or removed within an organization, though a default set of amenities is always present.
+ * Each amenity may then be associated or disassociated from any spaces within that organization.
+ */
+public class Amenity implements ApiResponseModel {
+
+    /**
+     * Constants
+     */
+
+    public static final String MIME_TYPE = "vnd.robinpowered.amenity.v1";
+
+
+    /**
+     * Properties
+     */
+
+    // Immutable
+    private final Integer id;
```

Comment:
Is there a reason why we have non-null properties as nullable objects?
Could this be `int id` instead?